server/src/test/java/io/crate/expression/reference/sys/SysShardsExpressionsTest.java

```diff
@@ -287,7 +287,6 @@ public void testPartitionIdent() throws Exception {
 
     @Test
     public void testPartitionIdentOfNonPartition() throws Exception {
-        // expression should return NULL on non partitioned tables
```

Comment:
Test has `assertThat(shardExpression.value()).isEqualTo("");` so this comment doesn't make sense